docs/index.adoc

```diff
@@ -38,69 +38,77 @@ The specification DSL makes it much easier to read the tests, and also includes
-
-- link:../test/untangled_spec/all_tests.cljs[all_tests.cljs] : An entry point for CI testing from the command line.
-- link:../dev/cljs/user.cljs[user.cljs] : The entry point for browser test rendering.
-- link:../test/untangled_spec/tests_to_run.cljs[tests_to_run.cljs] : A file that does nothing more than require all of the specs. The test runners search for
-testing namespaces, so if you don't load them somewhere, they won't be found. Since there are two places tests
-run from (browser and CI) it makes sense to make this DRY.
-- `*_spec.clj(s)` : Are spec files containing tests/assertions.
+=== CI Tests
 
 There is a top level link:../package.json[package.json] file for installing node packages to run CI tests.
 
-The link:../project.clj[project.clj] includes various things to make all of this work:
+* The https://github.com/bensu/doo#usage[lein doo plugin], for running tests through karma *via* node (in Chrome).
+* link:../test/untangled_spec/all_tests.cljs[all_tests.cljs] : An entry point for CI testing from the command line.
+* A `:doo` section to configure the CI runner
+* A cljsbuild with id `:automated-tests` is the CI tests output.
+
+=== Browser Tests
+
+* link:../dev/cljs/user.cljs[user.cljs] : The entry point for browser test rendering.
+* A cljsbuild with id `:test` is the browser test build.
 
-- The https://github.com/bensu/doo#usage[lein doo plugin], for running tests through karma *via* node (in Chrome).
-- A `:doo` section to configure the CI runner
-- A cljsbuild for testing with figwheel true. The build with id "test" link:../project.clj#L36[here], is the browser test build.
-- A cljsbuild for testing with lein doo. The build with id "automated-tests" link:../project.clj#L47[here], is the CI tests output.
-- The https://github.com/jakemcc/lein-test-refresh[lein test-refresh plugin], which will re-run server tests on save, and also can be configured with the spec renderer (see the `:test-refresh` section in the project file).
+=== Clojure Tests
+
+* The https://github.com/jakemcc/lein-test-refresh[lein test-refresh plugin], which will re-run server tests on save, and can be configured (see the `:test-refresh` section in the link:../project.clj[project.clj]).
+* A cljsbuild with id `:spec-renderer` for rendering server specs in the browser. +
+NOTE: you will not need to copy/have this build in your own project, see <<With reporting in the browser>>
+* link:../dev/clj/user.clj[user.clj] : The entry point for running clojure tests that should be rendered in the browser.
 
 == Running server tests
 
-To run all specs, just use `lein test-refresh` at the command line, +
+=== With reporting in the terminal
+Use `lein test-refresh` at the command line,
 and make sure https://github.com/jakemcc/lein-test-refresh[jakemcc/lein-test-refresh] is in your `:plugins`.
 
+=== With reporting in the browser
+
+* Create a started system using `untangled-spec.suite/test-suite` and manage it's lifecycle +
```

Comment:
It would be clearer to state what test-suite does, then use sub-items to enumerate details:

- `(untangled-spec.suite/test-suite args-description)` can be used to create a web server that will serve a test report renderer.
     - See docstring for further info on args
     - Minimum they need to know to make it work